src/storage/IndexRepresentationStore.ts

```diff
@@ -0,0 +1,58 @@
+import assert from 'assert';
+import type { Representation } from '../ldp/representation/Representation';
+import type { RepresentationPreferences } from '../ldp/representation/RepresentationPreferences';
+import type { ResourceIdentifier } from '../ldp/representation/ResourceIdentifier';
+import { NotFoundHttpError } from '../util/errors/NotFoundHttpError';
+import { isContainerIdentifier } from '../util/PathUtil';
+import type { Conditions } from './Conditions';
+import { cleanPreferences, matchesMediaType } from './conversion/ConversionUtil';
+import { PassthroughStore } from './PassthroughStore';
+import type { ResourceStore } from './ResourceStore';
+
+/**
+ * Returns an index representation when requests are made to a container.
+ * The index representation will be returned when the following conditions are fulfilled:
+ *  * The request targets a container.
+ *  * A resource with the given `indexName` exists in the container. (default: "index.html")
+ *  * The highest weighted preference matches the `mediaRange` (default: "text/html")
+ * Otherwise the request will be passed on to the source store.
+ * In case the index representation should always be returned when it exists,
+ * the `mediaRange` should be set to "\* / \*".
```

Comment:
yes, but then javascript thinks I'm closing the comment block 😅 . The `*` also disappear from the generated jsdoc since it's interpreted as a comment character hence the escaping. Would love to know if there is a way to just have `*/*` there.

Reply:
https://www.compart.com/en/unicode/U+2215 😉

Reply:
Or a zero-width space.